tests/Unit/CssInlinerTest.php

```diff
@@ -3140,6 +3140,96 @@ public function inlineCssNotInDebugModeIgnoresOnlyInvalidExcludedSelector(): voi
         self::assertStringContainsString('<p class="z"></p>', $result);
     }
 
+    /**
+     * @test
+     *
+     * @dataProvider excludedCssRuleDataProvider
+     *
+     * @param string $css
+     * @param array<string> $exludedSelectors
+     * @param string $expectedInlineCss
```

Comment:
Please let's not duplicate in PHPDoc what we already have in native type declarations.

Reply:
Consider the https://cs.symfony.com/doc/rules/phpdoc/no_superfluous_phpdoc_tags.html rule then :)